text/processing.py

```diff
@@ -280,16 +280,16 @@ class WriteToTextFileJob(Job):
     Write a given content into a text file, one entry per line
     """
 
-    def __init__(
-        self,
-        content,
-    ):
+    __sis_hash_exclude = {"out_name": "file.txt"}
+
+    def __init__(self, content, out_name: str = "file.txt"):
         """
         :param list|dict|str content: input which will be written into a text file
+        :param out_name: user specific name
```

Comment:
```suggestion
        :param out_name: user specific file name for the output file
```
just a tad more verbose